guestbook/comments/admin.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+
+from .models import Comment, Message
+
+class CommentAdmin(admin.ModelAdmin):
```

Comment:
If you don't define any attributes on the `CommentAdmin` or a `ModelAdmin` in general, you can just use `admin.site.register(Comment)`. :wink: